src/lib/process_crawl.js

```diff
@@ -221,9 +237,10 @@ function getMetrics(crawl_row) {
 }
 
 module.exports = function(crawl_row, log) {
+  if(!crawl_row) return undefined;
```

Comment:
yeah, that happens in the beginning of the times, when we get the first crawl and there is no a previous one for it, so process_crawl() is called for undefined and returns undefined